cw_evm/lib/evm_chain_transaction_info.dart

```diff
@@ -0,0 +1,87 @@
+// ignore_for_file: annotate_overrides, overridden_fields
+
+import 'dart:math';
+
+import 'package:cw_core/format_amount.dart';
+import 'package:cw_core/transaction_direction.dart';
+import 'package:cw_core/transaction_info.dart';
+
+class EVMChainTransactionInfo extends TransactionInfo {
```

Comment:
Adjusted the implementation so that won't be needed for this class anymore

Reply:
nope, the changes you made would cause some issues.
the token symbol is not specific for the wallet type, rather the literally the token of the transaction (ex. USDC, SHIBA, etc...)
and in the fee function you are using the token symbol, although the fee will be with the base currency of the network (i.e ETH for ethereum and Matic for polygon)

you can make it as abstract and make a separate getter for `feeCurrency` and this is the only thing that should be overridden in the child class for Ethereum and Polygon

Reply:
Alright,

Adjusted it to receive the `feeCurrency` from child classes